src/problems/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Ah ok nice! Sorry got it confused with the TestCase class for the problems themselves.